### PR TITLE
Changing from my guess on commit

### PR DESCRIPTION
**Pull request by Chris-Wales**
This pull request adds a folder and a file for Chris-Wales
@Chris-Wales Fill this out ASAP

### CODE REVIEW

File: Chris-Wales/USERNAME-Chris-Wales-Dallas.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Dallas
+
+### Restaurants
+
```
Comment (richarde002) [inline Chris-Wales/USERNAME-Chris-Wales-Dallas.md:5]: @Chris-Wales What are some good places to eat in Dallas?

**Comment by Chris-Wales** [comment]
@richarde002  my favorite BBQ is Lockharts.  They're fantastic and sell out of food almost every day!

File: Chris-Wales/USERNAME-Chris-Wales-Dallas.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Dallas
+
+### Restaurants
```
Comment (Shobhit1) [inline Chris-Wales/USERNAME-Chris-Wales-Dallas.md]: Hello from Dallas. 😄

Comment (Chris-Wales) [inline Chris-Wales/USERNAME-Chris-Wales-Dallas.md]: @Shobhit1  Hi!  Where abouts in Dallas are you?  I'm actually setup in Addison.

Comment (Shobhit1) [inline Chris-Wales/USERNAME-Chris-Wales-Dallas.md]: @Chris-Wales I am a student in UTD and live on campbell near the university

**Pull request merged**